pkg/asset/manifests/operators.go

```diff
@@ -225,3 +227,12 @@ func applyTemplateData(template *template.Template, templateData interface{}) []
 	}
 	return buf.Bytes()
 }
+
+// Load returns the manifests asset from disk.
+func (m *Manifests) Load(p asset.FilePatternFetcher) (found bool, err error) {
```

Comment:
Do we need to load the tectonic config data as well?

Reply:
We will load anything under `manifests/*` so that users can add their customized manifests there too.

Reply:
My point is that this asset writes the "00_cluster-config.yaml" in the tectonics directory, but it does not read that file back in on `Load`.